src/main/java/seedu/application/ui/HelpWindow.java

```diff
@@ -16,7 +16,17 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://ay2324s1-cs2103t-w12-3.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String HELP_MESSAGE_COMMANDS = "Here are the available commands and how to use them: \n\n"
+            + "Words in UPPER_CASE are the parameters to be supplied by the user\n\n"
+            + "1. To add an application: add n/NAME \n"
+            + "2. To delete an application: delete INDEX \n"
+            + "3. To list all applications: list \n"
+            + "4. To mark the status of an application: mark INDEX s/STATUS \n"
+            + "5. To set a deadline for an application: deadline INDEX d/DEADLINE \n"
+            + "6. To find an application by company: find -c [KEYWORDS] \n"
```

Comment:
Might be better to combine 6 and 7: "To find an application by field: find -FIELD [KEYWORDS]